Unsorted/24299388_172c77c.txt

```diff
@@ -9,13 +9,19 @@
 	<sjis>
 		今は商品の整理中なんだ<end_line>
 	</sjis>
+	<ascii>
+		We're organizing our stock today<end_line>
+	</ascii>
 </dialogue>
 <dialogue>
 	<info box="simple">
 	<portrait_l name="shop_clerk">
 	<sjis>
 		悪いけどまたあとで来てくれよ<end_line>
 	</sjis>
+	<ascii>
+		Sorry but please come at another time<end_line>
```

Comment:
Line is too long (38 chars vs 36 char limit)

Reply:
I forgot to mention but you can use as many lines as you need, You might add one to make it more readable. This goes to another text boxes too! just try to keep them at minimum :P

Reply:
This I ended up changing it more than I thought ( c0b273b ), it's like this now:
> Sorry, can you come at another time?<end_line>

It felt like the "please" was making it too polite when compared to the japanese one.
> 悪いけどまたあとで来てくれよ<end_line>

At the same  time I thought just writing as something like "Come at another a time." would be too straight so I ended up changing it to a question form.
I'm probably overthinking it so let me know what you think.